.golangci.example.yml

```diff
@@ -473,36 +519,51 @@ linters-settings:
+    # Forbid the use of the `exclude` directives.
+    # Default: false
     exclude-forbidden: false
 
   gomodguard:
     allowed:
-      modules:                                                        # List of allowed modules
-        # - gopkg.in/yaml.v2
-      domains:                                                        # List of allowed module domains
-        # - golang.org
+      # List of allowed modules.
+      modules:
+         - gopkg.in/yaml.v2
+      # List of allowed module domains.
+      domains:
+         - golang.org
     blocked:
-      modules:                                                        # List of blocked modules
-        # - github.com/uudashr/go-module:                             # Blocked module
-        #     recommendations:                                        # Recommended modules that should be used instead (Optional)
-        #       - golang.org/x/mod
-        #     reason: "`mod` is the official go.mod parser library."  # Reason why the recommended module should be used (Optional)
-      versions:                                                       # List of blocked module version constraints
-        # - github.com/mitchellh/go-homedir:                          # Blocked module with version constraint
-        #     version: "< 1.1.0"                                      # Version constraint, see https://github.com/Masterminds/semver#basic-comparisons
-        #     reason: "testing if blocked version constraint works."  # Reason why the version constraint exists. (Optional)
-      local_replace_directives: false                                 # Set to true to raise lint issues for packages that are loaded from a local path via replace directive
+      # List of blocked modules.
+      modules:
+        # Blocked module.
+        - github.com/uudashr/go-module:
+            # Recommended modules that should be used instead. (Optional)
+            recommendations:
+              - golang.org/x/mod
+            # Reason why the recommended module should be used. (Optional)
+            reason: "`mod` is the official go.mod parser library."
+      # List of blocked module version constraints.
+      versions:
+        # Blocked module with version constraint.
+        - github.com/mitchellh/go-homedir:
+            # Version constraint, see https://github.com/Masterminds/semver#basic-comparisons.
+            version: "< 1.1.0"
+            # Reason why the version constraint exists. (Optional)
+            reason: "testing if blocked version constraint works."
+      # Set to true to raise lint issues for packages that are loaded from a local path via replace directive.
+      local_replace_directives: false
 
   gosimple:
-    # Select the Go version to target. The default is '1.13'.
+    # Select the Go version to target.
+    # Default: 1.13
     go: "1.15"
     # https://staticcheck.io/docs/options#checks
     checks: [ "all" ]
```

Comment:
As far as I can tell, the default is `checks: [ "all", "-ST1000", "-ST1003", "-ST1016", "-ST1020", "-ST1021", "-ST1022", "-ST1023" ]`